KarhooUISDK/CommonUI/BottomSheet/KarhooBottomSheetViewModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  BottomSheetMPV.swift
+//  KarhooUISDK
+//
+//  Created by Diana Petrea on 03.11.2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+protocol BottomSheetViewModel {
+    var title: String { get set }
+    var cornerRadius: CGFloat { get set }
+    var backgroundColor: UIColor { get set }
+    var callback: ScreenResultCallback<AnyObject>? { get set }
+}
+
+class KarhooBottomSheetViewModel: BottomSheetViewModel {
+    var title: String = "Some Title"
+    var backgroundColor: UIColor = KarhooUI.colors.white
+    var cornerRadius: CGFloat = UIConstants.CornerRadius.xxLarge
+    var callback: ScreenResultCallback<AnyObject>?
```

Comment:
As I wrote in another comment: I think view models `callback` should be private, and it should be assessable using methods.